gn3/computations/correlation_matrix.py

```diff
@@ -74,29 +52,16 @@ def get_scree_plot_data(pca_obj):
     """get scree plot data for generating plot x=x_vals,y = perc_var"""
 
     perc_var = np.round(pca_obj.explained_variance_ratio_*100, decimals=1)
-    x_vals = list(range(1, len(perc_var)+1))
+
+    x_vals = [f"PC{val}" for val in list(range(1, len(perc_var)+1))]
```

Comment:
**suggestion:**:
# #
```suggestion
    x_vals = [f"PC{val}" for val in range(1, len(perc_var)+1)]
```

No need to convert the range object to a list since it's already an iterable...